[
    {
        "repo_name": "auth0/node-auth0",
        "CVE_ID": "CVE-2020-15125",
        "Problem_Type": "CWE-209",
        "Description": "In auth0 (npm package) versions before 2.27.1, a DenyList of specific keys that should be sanitized from the request object contained in the error object is used. The key for Authorization header is not sanitized and in certain cases the Authorization header value can be logged exposing a bearer token. You are affected by this vulnerability if you are using the auth0 npm package, and you are using a Machine to Machine application authorized to use Auth0's management API",
        "URL": "https://github.com/auth0/node-auth0/pull/507/commits/62ca61b3348ec8e74d7d00358661af1a8bc98a3c",
        "Tag": "['Patch', 'Third Party Advisory']",
        "fix_commit_hash": "62ca61b3348ec8e74d7d00358661af1a8bc98a3c",
        "inducing_commit_hash_pyszz": [
            "a186023560ab5509e486b53b5db6aad690407ad8"
        ]
    },
    {
        "repo_name": "auth0/node-auth0",
        "CVE_ID": "CVE-2020-15125",
        "Problem_Type": "CWE-209",
        "Description": "In auth0 (npm package) versions before 2.27.1, a DenyList of specific keys that should be sanitized from the request object contained in the error object is used. The key for Authorization header is not sanitized and in certain cases the Authorization header value can be logged exposing a bearer token. You are affected by this vulnerability if you are using the auth0 npm package, and you are using a Machine to Machine application authorized to use Auth0's management API",
        "URL": "https://github.com/auth0/node-auth0/pull/507",
        "Tag": "['Patch', 'Third Party Advisory']",
        "fix_commit_hash": "e23cd867a317c26b2aa39bd7fbacd59acb243195",
        "inducing_commit_hash_pyszz": []
    }
]